### ISSUE TITLE
Add common Rust crates

### ISSUE THREAD

**Issue by dhouck** [compiler-explorer/compiler-explorer#2515]
I'm new to Rust so can't make a list of all the ones people might expect.  I know I want:
 - [ ] https://crates.io/crates/contracts but don't expect most people to
 - [ ] https://crates.io/crates/rand, which seems likely to be relatively common.

**Comment by partouf** [comment]
Duplicate of #754 and #1925

**Comment by RubenRBS** [comment]
Hi, thanks for taking the time tu suggest us some useful crates!
As @partouf mentioned in the linked issues, this is something we currently don't support, but we're actively looking into (See releated issue). I've recorded your suggestion, and will close this as to try to keep everything crate releated in one place, but please, once support lands, feel free to add additional suggestions of crates :)
